### PR TITLE
add 'i' toggle for info

### PR DESCRIPTION
**Pull request by benjamink**
Pressing the `i` key will toggle the display of the info at the bottom of the screen.

### CODE REVIEW

File: src/common/show_info.c [hunk 1]
```diff
@@ -64,11 +64,12 @@ void show_info() {
 	}
 
 	GOTOXY(0, 23);
-	print_reverse("F");    cputs("reeze ");
-	print_reverse("R");    cputs("eset ");
+	print_reverse("F");    cputs("rz ");
```
Comment (benjamink) [inline src/common/show_info.c:67]: Making more room on screen by abbreviating these more

File: src/common/world.c [hunk 2]
```diff
@@ -105,6 +105,12 @@ void get_world_cmd() {
 					get_broadcast();
 					is_showing_broadcast = true;
 					break;
+				case 6: // enable info
```
Comment (benjamink) [inline src/common/world.c:108]: I don't really understand what is happening here.  Are these commands that can be accessed via an API GET?  What do the numbers mean?  I guessed.

File: src/common/data.s [hunk 3]
```diff
@@ -81,5 +82,6 @@ _is_running_sim:        .byte $01
 _current_step:          .byte $ff
 _info_display_count:    .byte $00
 _is_darkmode:           .byte $00
+_is_showing_info:       .byte $01
```
Comment (benjamink) [inline src/common/data.s:85]: I guessed at this assuming `$01` is setting the item to `true` (`1`).

**Comment by benjamink** [comment]
This is the most recent working build: https://fujinet.skdev.org/personal/superbenk/bouncy-world/

I haven't tested on other platforms.

File: src/common/data.s (hunk 3 above)
Comment (markjfisher) [reply src/common/data.s:85]: it is

File: src/common/world.c (hunk 2 above)
Comment (markjfisher) [reply src/common/world.c:108]: yes, these are the codes that I would need to add to the server to send the client a command that it should turn on/off its info
However, it means I need to implement it in the server too now to work!

Comment (benjamink) [reply src/common/world.c:108]: Where is the server code?  I can take a look at adding that functionality.

Comment (benjamink) [reply src/common/world.c:108]: Assuming this? https://github.com/markjfisher/bounce-world

Comment (benjamink) [reply src/common/world.c:108]: As noted on Discord - looked into this but it's a bit over my head currently.  Is this a blocker to merging this change?  If so would it help to just remove the changes to `world.c` for now & come back to exposing the function as an API endpoint later?

**Comment by markjfisher** [comment]
I'm going to accept this, I have the fix for it, but I need write permissions to  your repo. Easier will be simply to add my fix on top of your changes.

**Pull request merged**